### ISSUE TITLE
Affected line numbers missing

### ISSUE THREAD

**Issue by LionelMarbot** [sebastianbergmann/diff#33]
Hi,

When creating a diff of following two files, the affected line numbers are not being added to the diff:

Original: https://testing.enuvo.ch/diff/a.txt
New: https://testing.enuvo.ch/diff/b.txt

Here are the first couple of lines from the diff:

```
--- Original
+++ New
@@ @@
- <a contenteditable="true">Analisi risultati</a>
+ <a contenteditable="true">Analizza risultati</a>

@@ @@
- <p contenteditable="true">Puoi iniziare a creare il tuo sondaggio online subito dopo la tua registrazione gratuita. La nostra interfaccia facile da usare e che non ha bisogno di spiegazioni ti aiuta a costruire questionari meravigliosi in pochi minuti. Puoi scegliere tra una varietà di diverse tipologie di domanda, componi le tue proprie domande, aggiungi testo esplicativo, carica immagini e molto altro; di conseguenza progettare sistematicamente il tuo sondaggio un pezzo alla volta. Sarai sempre in grado di vedere un'anteprima esatta di domande singole come pure il tuo intero questionario. Inoltre, puoi modificare il tema del tuo questionario utilizzando i tuoi propri colori e logo. Prenditi tutto il tempo necessario per creare i tuoi questionari online - non vi sono limiti di tempo.</p>
+ <p contenteditable="true">Puoi iniziare a creare il tuo sondaggio online subito dopo la tua registrazione gratuita. La nostra interfaccia, facile da usare e che non ha bisogno di spiegazioni, ti aiuta a costruire questionari meravigliosi in pochi minuti. Puoi scegliere tra una varietà di diverse tipologie di domanda, comporre le tue domande, aggiungere testo esplicativo, caricare immagini e molto altro; e, di conseguenza, progettare sistematicamente il tuo sondaggio un pezzo alla volta. Sarai sempre in grado di vedere un'anteprima esatta di domande singole come anche il tuo intero questionario. Inoltre, puoi modificare il tema del tuo questionario utilizzando i tuoi colori e il tuo logo. Prenditi tutto il tempo necessario per creare i tuoi questionari online - non ci sono limiti di tempo.</p>
```

As you can see, there is no info between the `@@ @@`.

Any idea what is going on?

Thanks,
Lionel

**Comment by SpacePossum** [comment]
Hi @sebastianbergmann 

Over @ PHP-CS-Fixer we get requests for line numbers to be added to the diff every now and than (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/299 and others).
Since we are using this repo (thanks for that!) as a base to generate the diff's I was wondering if you would be interested in having this feature to be added here (only showing when configured of course).

**Comment by sebastianbergmann** [comment]
I would consider merging a pull request that adds line numbers to `@@ @@`, of course. I just don't have time to implement this myself.

**Comment by sebastianbergmann** [comment]
Implemented by @SpacePossum in #64.
